src/backend/distributed/sql/citus--9.5-1--10.0-1.sql

```diff
@@ -7,3 +7,29 @@ DROP FUNCTION IF EXISTS pg_catalog.citus_total_relation_size(regclass);
 #include "udfs/citus_finish_pg_upgrade/10.0-1.sql"
 
 #include "../../columnar/sql/columnar--9.5-1--10.0-1.sql"
+
+-- Remove all the appended shardId from the names of constraints of partitioned tables,
+-- with the eception of foreign key constraints that should keep the shardId in their names
```

Comment:
```suggestion
-- with the exception of foreign key constraints that should keep the shardId in their names
```

Reply:
Thanks for the quick fixes on my typos. (Broken keyboard + I never learned how to type ~seperate~ separate)